app/helpers/application_helper.rb

```diff
@@ -43,7 +43,7 @@ def render_attribute_content(attribute)
     render(partial: "quick_reference/#{attribute}", formats: [:md]).to_json.html_safe
   end
 
-  def logo_image_url
+  def logo_image_path
```

Comment:
Just out of curiosity: Why did these need to be updated?

Reply:
The `_url` suffix in rails outputs absolute URLs and unfortunately when scraping a local version of the site it meant the host was being output as localhost:3000. By using relative URLs I was able to output compatible markup.